packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -203,10 +203,20 @@ export const useProductFullDetail = props => {
             appendOptionsToPayload(payload, optionSelections, optionCodes);
         }
 
+        let addItemMutation;
+
+        // Prepare the params to add and use the proper mutation.
+        if (productType === 'SimpleProduct') {
```

Comment:
`productType` is `undefined` in this scope, bad copy/paste from the action.

Reply:
Ooops!